src/main/java/com/vmtapp/enterprise/dao/PhotoDAO.java

```diff
@@ -0,0 +1,35 @@
+package com.vmtapp.enterprise.dao;
+
+import com.vmtapp.enterprise.dto.Photo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+@Repository
+public class PhotoDAO implements IPhotoDAO {
+
+    @Autowired
+   // private PhotoRepository photoRepository;
+
+    @Override
+    public void save(Photo photo) {
+        //photoRepository.save(photo);
+    }
+
+    @Override
+    public void saveImage(MultipartFile imageFile, Photo photo) throws IOException {
+        Path currentPath = Paths.get(".");
+        Path absolutePath = currentPath.toAbsolutePath();
+        photo.setPath(absolutePath + "/src/main/resources/static/photos/");
+        byte[] bytes = imageFile.getBytes();
+        Path path = Paths.get(photo.getPath() + imageFile.getOriginalFilename());
+        Files.write(path, bytes);
+
+    }
+
+}
```

Comment:
Moving forward we can utilize interface for the photos

Reply:
This appears to be an incomplete feature.  In a code review, focus on reducing technical debt of existing code.  Avoid adding new, incomplete functionality .
1) Only features that meet the Definition of Done should be merged to Master.
2) Unused code is clutter, which increases technical debt.